### ISSUE TITLE
Looser semantic-release  peer dependency version

### ISSUE THREAD

**Issue by mario-jerkovic** [yllieth/semantic-release-ms-teams#27]
Hi 👋,

With latest **semantic-release@19.0.2** when installing this plugin with npm I get following error:

```
Could not resolve dependency:
npm ERR! peer semantic-release@">=17.0.0 <18.0.0" from semantic-release-ms-teams@1.2.1
npm ERR! node_modules/semantic-release-ms-teams

```

This happens because semantic-release peer dependency version is to strict, [link](https://github.com/yllieth/semantic-release-ms-teams/blob/main/package.json#L46)

I would suggestion something like this:

```json
{
 "peerDependencies": {
    "semantic-release": "^18 || ^19",
  },
}
```

**Comment by yllieth** [comment]
Hi @mario-jerkovic. Thank you for reporting this, and using the plugin. I will fix it as soon as possible.

**Comment by MatejVukosav** [comment]
@yllieth any idea when fix will be available :)

**Comment by yllieth** [comment]
Hi @MatejVukosav, @mario-jerkovic. I'm really sorry it took so long. I recently published a new version `2.1.0` that supports semantice-release@19, as well as dropping support of node15 (as semrel did in 19). It also adds unit tests and a couple of refactorings. I hope it will work for you guys.
